Makefile

```diff
@@ -160,14 +160,11 @@ start-node-with-flamegraph: rm-test-db ## 🚀🔥 Starts an ethrex client used
 	--dev \
 	--datadir test_ethrex
 
-load-test: install-cli ## 🚧 Runs a load-test. Run make start-node-with-flamegraph and in a new terminal make load-node
-	ethrex_l2 test load --path test_data/private_keys.txt -i 1000 -v  --value 100000
+load-test: ## 🚧 Runs a load-test. Run make start-node-with-flamegraph and in a new terminal make load-node
+	cargo run --manifest-path ./cmd/load_test/Cargo.toml -- -k ./test_data/private_keys.txt -t eth-transfers
```

Comment:
We may want to:
1. Precompile, since we sometimes use `time` with the command to get an idea of whether a change improved performance for this and the building will introduce too much noise
2. Use `--release` to spam the client more effectively